packages/mdc-checkbox/index.ts

```diff
@@ -23,7 +23,7 @@
 
 import {getCorrectEventName} from '@material/animation/index';
 import MDCComponent from '@material/base/component';
-import {EventType, SpecificEventListener} from '@material/dom/index';
+import {EventType, SpecificEventListener} from '@material/base/types';
```

Comment:
I think we should also be importing/exporting from @material/base/index

Reply:
Done